app/build.gradle

```diff
@@ -9,12 +9,12 @@ if (keystorePropertiesFile.exists()) {
 }
 
 android {
-    compileSdkVersion 31
+    compileSdkVersion 32
```

Comment:
do we really need this? None app has it yet

Reply:
I will revert

Reply:
Done

Reply:
I forgot that we needed it for the latest  version of CameraX

Reply:
Will revert the CameraX version to the more stable [1.1.0-rc02](https://developer.android.com/jetpack/androidx/releases/camera#camera-lifecycle-1.1.0-rc02)